content/citation-tags.tsv

```diff
@@ -1,4 +1,5 @@
 tag	citation
+techblog-csl	url:http://blogs.nature.com/naturejobs/2017/05/03/techblog-create-the-perfect-bibliography-with-the-csl-editor/
```

Comment:
In a follow-up commit, I want to switch underscores in tag names to `-`, as the later is valid in strict pandoc formatting, and therefore should be preferred.

Reply:
Done in https://github.com/greenelab/meta-review/pull/102